ui/src/alerting/components/CheckEOHeader.tsx

```diff
@@ -47,10 +79,17 @@ const CheckEOHeader: FC<Props> = ({name, onSetName, onCancel, onSave}) => {
     }
   }
 
-  const saveButtonStatus =
-    saveStatus === RemoteDataState.Loading
-      ? ComponentStatus.Loading
-      : ComponentStatus.Default
+  const saveButtonStatus = () => {
```

Comment:
how come this is a function?

Reply:
I think it reads much nicer as a function.